geniza/corpus/sitemaps.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.sitemaps import Sitemap
+
+from geniza.corpus.models import Document
+
+
+class DocumentSitemap(Sitemap):
+    def items(self):
+        return Document.objects.filter(status=Document.PUBLIC)
+
+    def lastmod(self, obj):
+        return obj.last_modified
+
+
+class DocumentScholarshipSitemap(Sitemap):
+    def items(self):
+        return Document.objects.filter(status=Document.PUBLIC, footnotes__isnull=False)
```

Comment:
Do we 404 on document scholarship records pages that don't have any footnotes? They should only be excluded if the page doesn't exist. (But we could set the priority very low if they don't have footnotes.)

Reply:
Yes there is currently a 404 when there are no footnotes:
https://test-geniza.cdh.princeton.edu/en/documents/516/
https://test-geniza.cdh.princeton.edu/en/documents/516/scholarship/

Reply:
Ok, great — I missed where in the view logic it was excluding documents with no footnotes. So this logic is sound. (But add a comment to make it explicit)

Reply:
fedabc082b8e3ecaaa6b10a9667cb6adfeee28bf ✅